packages/wallet/anypay-sdk/src/widget/components/SendForm.tsx

```diff
@@ -109,9 +114,35 @@ export const SendForm: React.FC<SendFormProps> = ({
   onComplete,
   account,
   sequenceApiKey,
+  apiUrl,
+  env,
 }) => {
   const [amount, setAmount] = useState('')
+  const [recipientInput, setRecipientInput] = useState('')
   const [recipient, setRecipient] = useState('')
+  const {
+    data: ensAddress,
+    isLoading,
+    error,
+  } = useEnsAddress({
```

Comment:
Probably may have to point to `chainId: 1` (if that's still the case)

Reply:
good catch!